openetl_utils/main_storage_class.py

```
@@ -0,0 +1,175 @@
+import os
+from abc import ABC, abstractmethod
+
+from openetl_utils import install_libraries
+from openetl_utils.enums import ConnectionType, AuthType
```

Ruff (0.12.2)

`openetl_utils.enums.AuthType` imported but unused

Remove unused import: `openetl_utils.enums.AuthType`

(F401)